### PR TITLE
Player join

### PR DESCRIPTION
**Pull request by chelsea0x3b**

### CODE REVIEW

**Comment by chelsea0x3b** [comment]
[Handle PlayerJoin in Game](https://trello.com/c/5bYtigbg/11-handle-playerjoin-in-game)

**Comment by chelsea0x3b** [comment]
[Handle JoinNotification](https://trello.com/c/Zkopvu65/25-handle-joinnotification)

**Review by NebuchadnezzarXIII — APPROVED** [review]
Only thing that's a little worrisome is that we have to overwrite an initial tag instead of providing the tag to begin with, but given the nature of websockets I don't think this can be helped.

**Comment by chelsea0x3b** [comment]
Yeah Jason those are my thoughts exactly

File: frontend/clueless.js
Comment (swheaton) [inline frontend/clueless.js:26]: Don't we want to keep the block that checks if the JSON parses correctly?

Comment (chelsea0x3b) [reply frontend/clueless.js:26]: true we should. previously this was getting overridden anyways by the code on line 100.

**Comment by swheaton** [comment]
You can go ahead and merge this branch once the conflicts are resolved. I was going to resolve the conflicts myself but I wanted to build and test before merging, and I don't have a machine to test on yet

**Pull request merged**
